src/uncontrast/pages/UploadNewPaper/ProgressComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { uploadPaperPageTopSection } from "../../../Utils/HardCoded";
+
+export default function ProgressComponent({ status, paperNmae, experiment }) {
+  const Arrow = (
```

Comment:
Maybe put this outside the component so it wouldn't create a new icon every render